sdk/personalizer/Azure.AI.Personalizer/src/Models/PersonalizerClient.cs

```diff
@@ -16,23 +16,27 @@ namespace Azure.AI.Personalizer
     ///<see href="https://docs.microsoft.com/azure/cognitive-services/personalizer/what-is-personalizer"/>
     public class PersonalizerClient
     {
-        private readonly ClientDiagnostics _clientDiagnostics;
-        private readonly HttpPipeline _pipeline;
-        private readonly bool _isLocalInference;
+        private readonly ClientDiagnostics clientDiagnostics;
+        private readonly HttpPipeline pipeline;
+        private readonly bool isLocalInference;
         private string stringEndpoint;
-        private string apiKey;
+        private AzureKeyCredential azureKeyCredential;
+        private TokenCredential tokenCredential;
+        private int liveModelRefreshTimeInMinutes = 15;
+        private DateTimeOffset tokenExpiry;
+        private DateTimeOffset liveModelLastRefresh;
+        private string[] scopes = { "https://cognitiveservices.azure.com/.default" };
         private float subsampleRate = 1.0f;
 
-        private readonly RankProcessor _rankProcessor;
-
+        private Lazy<RankProcessor> _rankProcessor;
```

Comment:
Thank you for fixing those member variable names! Can you fix this name also? It should be a quick operation via VS.